config/config.mainnet.json

```diff
@@ -91,6 +93,17 @@
           "callbacks": {
             "Ronin": "BridgeOperatorsApprovedCallback"
           }
+        },
+        "ExternalCallFailedSubscription": {
+          "to": "0x3Fb325b251eE80945d3fc8c7692f5aFFCA1B8bC2",
+          "type": 1,
+          "handler": {
+            "contract": "BridgeTracking",
```

Comment:
Why above is BridgeReward and here is BridgeTracking

Reply:
The event is emitted in BridgeTracking contract but we need to create transaction to BridgeReward contract